packages/eclipse/src/styles/globals.css

```
@@ -4,746 +4,388 @@
+  --spacing-sm: 1.25rem;
+  --spacing-md: 1.5rem;
+  --spacing-lg: 1.75rem;
+  --spacing-xl: 2rem;
+  --spacing-2xl: 2.25rem;
+  --spacing-3xl: 2.5rem;
+  --spacing-4xl: 3rem;
+  --spacing-5xl: 4rem;
+  --spacing-6xl: 6rem;
+
+  /* Typography - Font Families */
+  --font-sans-display: "Mona Sans", Inter, Roboto, "Helvetica Neue", "Arial Nova", "Nimbus Sans", Arial, sans-serif;
```

Stylelint (17.3.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)